src/inverseRobot.js

```diff
@@ -24,7 +24,23 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const goodRobot = {};
+  let isWrongRobot = false;
+
+  for (const key in robot) {
+    if (goodRobot.hasOwnProperty(robot[key])) {
```

Comment:
you use "robot[key]" several times so you should add this in variable

Reply:
Variables are needed for the result of an expression that is repeated or for the result of a function that needs to be used multiple times.

robot[key] is a variable (link, pointer) that contains the address of the key property of the robot object. 

Why assign one variable the value of another variable?